src/ApplicationInsights.Kubernetes/Debuggings/KubernetesDebuggingServiceCollectionBuilderFactory.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.ApplicationInsights.Kubernetes.Debugging
+{
+    public sealed class KubernetesDebuggingServiceCollectionBuilderFactory
```

Comment:
Is this class being used? I do not see any reference to it, maybe I am missing something...

Reply:
No, it is not referenced. The intention of this class is to provide the end user a way to inject some fake data into their telemetry. This will only be used in the users' code when they prefer to.

I am not advertising this because I am not 100% sure the value for this. If no one asks (except you), I might just remove it later.

Reply:
:/ 

I am not a big fan of adding features just in case someone *might* need it, but I will let you make the final call.

Reply:
More and more I am leaning toward to agree with you.
Not doing this for this PR since it is out of scope, I will seriously consider it.